### ISSUE TITLE
[Bug] [ErrorException] exclude_calendar is deprecated.

### ISSUE THREAD

**Issue by sidorov89** [temporalio/sdk-php#522]
### What are you really trying to do?
I'm trying add workflow into Temporal Schedule

### Describe the bug

```
#[AsCommand(name: 'check:status')]
final class CheckStatusCommand extends Command
{

    public function __invoke(): int
    {
              $client = \Temporal\Client\ScheduleClient::create(
                  \Temporal\Client\GRPC\ServiceClient::create('localhost:7233'),
              );
      
              $handle = $client->createSchedule(
                  Schedule\Schedule::new()->withAction(
                      Schedule\Action\StartWorkflowAction::new('WebsiteStatusWorkflow')
                          ->withRetryPolicy(\Temporal\Common\RetryOptions::new()->withMaximumAttempts(3))
                          ->withHeader(['foo' => 'bar'])
                          ->withWorkflowExecutionTimeout('40m')
                  )->withSpec(
                      Schedule\Spec\ScheduleSpec::new()
                          ->withIntervalList(5 * 60) // every 5 minutes
                          ->withJitter(60) // with jitter of 1 minute
                  ),
                  scheduleId: 'my-schedule-id',
              );
      }
}
```
After run code `php app.php check:status`  in console I get Exception: 
"[ErrorException]
exclude_calendar is deprecated.
in vendor/roadrunner-php/roadrunner-api-dto/generated/Temporal/Api/Schedule/V1/ScheduleSpec.php:355"

### Environment/Versions

- Roadrunner Version version: 2024.2.1,
- Temporal CLI 1.1.1 (Server 1.25.1, UI 2.31.2)
- OS and Platform Mac Book Intel Core i7
- Are you using Docker or Kubernetes or building Temporal from source? "brew install temporal"
- Framework Spiral 3.14.6

**Comment by corrosive4354** [comment]
Make sure all the related libraries you use (such as Roadrunner and Temporal) are the latest versions. You can try to update these dependencies to see if this deprecation warning has been resolved.

Seeing that you are using Docker, I would like to recommend [Servbay](https://servbay.com) to you, which is more professional and easy to use than Docker on Mac.

**Comment by roxblnfk** [comment]
@sidorov89 hi ✋ 
Could you check that PR fixes the deprecation error

**Comment by roxblnfk** [comment]
Resolved by adding PHP `protobuf` extension.
Other deprecations were fixed in #523
